### ISSUE TITLE
Mention python 2.7 as supported

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4650]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#4650|
|Type|enhancement|
|Created|2010-09-11 21:29:08Z|
|Branch|https://github.com/twisted/twisted/tree/2.7-support-4650|

We have a green builder, 2.7 has been release, so we should update the README and INSTALL files

<details><summary>Searchable metadata</summary>

```
trac-id__4650 4650
type__enhancement enhancement
reporter__therve therve
priority__high high
milestone__Twisted_10_2 Twisted 10.2
branch__branches_2_7_support_4650 branches/2.7-support-4650
branch_author__therve therve
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1284240548000000 1284240548000000
changetime__1284244549000000 1284244549000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve commented|
|-|-|

(In [30004]) Branching to '2.7-support-4650'

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph set owner to @therve|
|-|-|

Looks great.  Do it.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set status to `closed`|
|-|-|

(In [30006]) Merge 2.7-support-4650

Author: therve
Reviewer: glyph
Fixes: [#4650](https://github.com/twisted/twisted/issues/4650)

Mention Python 2.7 as a supported release in INSTALL and README.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
